tests/Agent/IntegrationTests/IntegrationTests/Logging/ContextDataNotSupportedTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using MultiFunctionApplicationHelpers;
+using NewRelic.Agent.IntegrationTestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace NewRelic.Agent.IntegrationTests.Logging.ContextData
+{
+    public abstract class ContextDataNotSupportedTestsBase<TFixture> : NewRelicIntegrationTest<TFixture>
+        where TFixture : ConsoleDynamicMethodFixture
+    {
+        private readonly TFixture _fixture;
+        private LoggingFramework _loggingFramework;
+        private bool _contextDataEnabled;
+
+        private const string InfoMessage = "HelloWorld";
+
+        // There are several entries in this dictionary to allow for different methods of adding the values in the test adapter
+        // If you need more entries for your framework, add them
+        private Dictionary<string, string> _expectedAttributes = new Dictionary<string, string>()
+        {
+            { "mycontext1", "foo" },
+            { "mycontext2", "bar" },
+            { "mycontext3", "test" },
+            { "mycontext4", "value" },
+        };
+
+
+        public ContextDataNotSupportedTestsBase(TFixture fixture, ITestOutputHelper output, bool contextDataEnabled, LoggingFramework loggingFramework) : base(fixture)
+        {
+            _fixture = fixture;
+            _loggingFramework = loggingFramework;
+            _contextDataEnabled = contextDataEnabled;
```

Comment:
All the test cases pass false to `contextDataEnabled` in the ctor. We should add test coverage for the new logger impl here, or in `ContextDataTests.cs`

Reply:
I removed the `contextDataEnabled` parameter as it's not necessary for this specific test case; context data will be explicitly enabled for the tests